hoot-services/src/main/java/hoot/services/command/ExternalCommandRunnerImpl.java

```diff
@@ -205,6 +207,15 @@ protected void processLine(String line, int level) {
         catch (Exception e) {
             exitCode = CommandResult.FAILURE;
             exception = e;
+
+            // Need to perform cleanup for import jobs
+            int jobType = DbUtils.getJobTypeByJobId(jobId);
+            if (jobType == JobType.IMPORT.ordinal() || jobType == JobType.CONFLATE.ordinal() || jobType == JobType.CLIP.ordinal()) {
```

Comment:
Maybe we could accomplish this by checking for resource id on the job?  That is an indication that the job wrote output to the hootpaidb

Reply:
yup. much better indicator